rpc/scanner/service.proto

```diff
@@ -25,9 +25,8 @@ message Licenses {
 message ScanOptions {
   repeated string       vuln_type          = 1;
   repeated string       scanners           = 2;
-  bool                  list_all_packages  = 3;
-  map<string, Licenses> license_categories = 4;
-  bool                  include_dev_deps   = 5;
```

Comment:
The number should not be changed. We should use `reserve`.
https://protobuf.dev/programming-guides/proto3/#assigning

Reply:
I doubted this decision, but decided to think about it tomorrow. Thank you for saving me time!
Fixed in https://github.com/aquasecurity/trivy/pull/5255/commits/6b6a6cc9dabd301a85d912eb4d542081ee2ce8eb